x-pack/plugins/security/public/account_management/account_management_app.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { CoreSetup, AppMountParameters } from 'src/core/public';
+import { AuthenticationServiceSetup } from '../authentication';
+import { UserAPIClient } from '../management';
+
+interface CreateDeps {
+  application: CoreSetup['application'];
+  authc: AuthenticationServiceSetup;
+  getStartServices: CoreSetup['getStartServices'];
+}
+
+export const accountManagementApp = Object.freeze({
+  id: 'security',
+  create({ application, authc, getStartServices }: CreateDeps) {
+    application.register({
+      id: this.id,
+      title: i18n.translate('xpack.security.account.breadcrumb', {
+        defaultMessage: 'Account Management',
+      }),
+      // TODO: switch to proper enum once https://github.com/elastic/kibana/issues/58327 is resolved.
```

Comment:
__note:__ just in case this issue isn't resolved before this PR is reviewed - Jest test tests that our magic number `3` corresponds to the `AppNavLinkStatus.hidden`